core/shared/src/main/scala/zio/ZIOAspect.scala

```diff
@@ -0,0 +1,78 @@
+package zio
+
+import zio.clock.Clock
+import zio.duration._
+import zio.internal.Executor
+
+import scala.concurrent.ExecutionContext
+
+trait ZIOAspect[+LowerR, -UpperR, +LowerE, -UpperE, +LowerA, -UpperA] {
+  def apply[R >: LowerR <: UpperR, E >: LowerE <: UpperE, A >: LowerA <: UpperA](zio: ZIO[R, E, A]): ZIO[R, E, A]
+}
+
+object ZAspect {
```

Comment:
Companion object should be the same name as the trait.

Reply:
Fixed now!